packages/gatsby/src/redux/reducers/index.js

```diff
@@ -12,6 +12,7 @@ import { webpackCompilationHashReducer } from "./webpack-compilation-hash"
 import { reducer as logReducer } from "gatsby-cli/lib/reporter/redux/reducer"
 import { lastAction } from "./last-action"
 import { jobsV2Reducer } from "./jobsv2"
+import { componenents } from "./components"
```

Comment:
Tests might fail because this named import should return `undefined` because of a typo.
Also, to improve the consistency with other reducers, could we rename the named export `componentsReducer` in this file, and in the `gatsby/src/redux/reducers/components.ts` file?

```suggestion
import { components } from "./components"
```

Reply:
Ouch, didn't notice, thank you!